README.md

```diff
@@ -1,2 +1,42 @@
 # crispy-api
 Typescript GraphQL API for `crispy-web`
+
+API based on [`Apollo`](https://www.apollographql.com/docs/apollo-server/) express server. 
+
+## How to use
```

Comment:
Need to specify somewhere that this is developers guide to setup dev environment

We also need a root do user docs